cypress/e2e/shared/secrets.test.ts

```diff
@@ -1,131 +1,230 @@
 import {Organization} from '../../src/types'
 
 describe('Secrets', () => {
-  beforeEach(() => {
-    cy.flush()
-    cy.signin()
-    cy.get('@org').then(({id}: Organization) =>
-      cy.fixture('routes').then(({orgs}) => {
-        cy.visit(`${orgs}/${id}/settings/`)
-      })
-    )
-    cy.getByTestID('tree-nav')
-    cy.getByTestID('secrets--tab').click()
-  })
+  describe('settings page', () => {
```

Comment:
I did not change this `settings page` test at all.
Only made it be a separate in tests, versus secrets test in notebooks (or other applications).